setup/kubernetes/k3s.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "k3s"
+description: Set up a Coder deployment with a Ubuntu + k3s
+---
+
+k3s is a lightweight Kubernetes distribution that works great with single-node
+or multi-node clusters alike.
+
+> This guide only covers installation on a new Ubuntu 20.04 LTS machine. If you
+> are looking to install on a local machine or an existing host,
+> [Kind](./kind.md) or k3d may be a better choice.
+
+## Prerequisites
+
+Before proceeding, please make sure that you have the following:
+
+1. **Ubuntu 20.04 machine**: This can be a bare metal or virtual machine. For
+   cloud compute, AWS EC2, GCP Compute Engine, Azure VMs, Vultr, or DigitalOcean
+   all work. Make sure the machine specs satisfies Coder's
```

Comment:
"all work" isn't great wording, but I think it is important to mention that cloud providers work well in this case